java-monitoring/run_protoc.sh

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env bash
+
+set -e
+
+LIBRARY_GEN_OUT=$(pwd)/../library_gen_out
+
+mkdir -p "${LIBRARY_GEN_OUT}"/google/monitoring/v3
+
+##################### Section 0 #####################
+# prepare tooling
+#####################################################
+# proto files from googleapis repository
+REPO_ROOT=$(pwd)/..
+cd "${REPO_ROOT}"
+
+if [ ! -d googleapis ]; then
+  git clone https://github.com/googleapis/googleapis.git
+fi
+
+GOOGLEAPIS_ROOT=${REPO_ROOT}/googleapis
+cd "${GOOGLEAPIS_ROOT}"
+git checkout 53a0be29c4a95a1d3b4c0d3a7a2ac8b52af2a3c0
```

Comment:
Can we extract this commit hash into a variable at the top?

Reply:
Yes, this is done in a more generic shell script (library_generation/generate_library.sh).

Reply:
Do we really need a script generating a script though?

Reply:
No, the generic shell script will execute protoc directly.